iceoryx2-bb/cxx/tests/src/raw_byte_storage_tests.cpp

```diff
@@ -129,9 +129,9 @@ TEST_F(RawByteStorageFixtureStrict, copy_constructor_copies_all_elements) {
     int32_t const tracking_id1 = 100;
     int32_t const tracking_id2 = 200;
     int32_t const tracking_id3 = 300;
-    obj.emplace_back(tracking_id1);
-    obj.emplace_back(tracking_id2);
-    obj.emplace_back(tracking_id3);
+    obj.emplace_back(static_cast<uint32_t>(tracking_id1));
+    obj.emplace_back(static_cast<uint32_t>(tracking_id2));
+    obj.emplace_back(static_cast<uint32_t>(tracking_id3));
```

Comment:
Can you check if the changes in this file are necessary? To met the casts seem superfluous.